go/keybase/main.go

```diff
@@ -115,6 +115,9 @@ func mainInner(g *libkb.GlobalContext) error {
 	checkSystemUser(g.Log)
 
 	if !cl.IsService() {
+		if err = logger.SaveConsoleMode(); err == nil {
```

Comment:
I thought it should only be used to gate the call to RestoreConsoleMode. Maybe I either don't need to use err, or could make one with a different name.